File: Backend/VestTour.Domain/Entities/Lining.cs
﻿using System;
using System.Collections.Generic;

namespace VestTour.Domain.Entities;

public partial class Lining
{
    public int LiningId { get; set; }

    public string? LiningName { get; set; }

    public string? ImageUrl { get; set; }

    public string? Status { get; set; }

    public virtual ICollection<Product> Products { get; set; } = new List<Product>();
}

File: Mobile/mobileapp/lib/models/lining.dart
class Lining {
  int? liningId;
  String? liningName;
  String? imageUrl;
  String? status;

  Lining({this.liningId, this.liningName, this.imageUrl, this.status});

  Lining.fromJson(Map<String, dynamic> json) {
    liningId = json['liningId'];
    liningName = json['liningName'];
    imageUrl = json['imageUrl'];
    status = json['status'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['liningId'] = this.liningId;
    data['liningName'] = this.liningName;
    data['imageUrl'] = this.imageUrl;
    data['status'] = this.status;
    return data;
  }
}
